src/Views/Attachment/ViewAttachment.js

```diff
@@ -152,6 +166,17 @@ const __class = declare('crm.Views.Attachment.ViewAttachment', [Detail, _LegacyS
   downloadingTemplate: new Simplate([
     '<li class="list-loading-indicator"><div>{%= $.downloadingText %}</div></li>',
   ]),
+  onTransitionTo: function onTransitionTo() {
+    const _renderFn = () => {
+      if (this.pdfDoc) {
+        this.pdfScale = 1;
+        this.renderPdfPage(this.pdfCurrentPage);
+      }
+    };
+
+    this._orientationHandle = connect.subscribe('/app/setOrientation', this, _renderFn);
```

Comment:
We might need to do one of two things here: Either disconnect this handle in the "onTransitionAway" lifecycle, or check for the existence of the handle and return early at the start of onTransitionTo.

Reply:
Sounds good, thanks for the feedback.